test/integration/shoots/creation/create_test.go

```diff
@@ -0,0 +1,224 @@
+// Copyright (c) 2019 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+	Overview
+		- Tests the creation of a shoot
+
+	BeforeSuite
+		- Parse Shoot from example folder and provided flags
+
+	Test: Shoot creation
+	Expected Output
+		- Successful reconciliation after Shoot creation
+ **/
+
+package creation
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"github.com/ghodss/yaml"
+	"time"
+
+	. "github.com/gardener/gardener/test/integration/shoots"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/sirupsen/logrus"
+
+	gardencorev1alpha1 "github.com/gardener/gardener/pkg/apis/core/v1alpha1"
+	"github.com/gardener/gardener/pkg/logger"
+	. "github.com/gardener/gardener/test/integration/framework"
+)
+
+var (
+	// flags
+	gardenerKubeconfigPath   = flag.String("gardener-kubecfg-path", "", "the path to the gardenerKubeconfigPath of the Garden cluster. File must exist.")
+	shootKubeconfigPath      = flag.String("shoot-kubecfg-path", "", "the path to where the Kubeconfig of the Shoot cluster will be downloaded to.")
+	seedKubeconfigPath       = flag.String("seed-kubecfg-path", "", "the path to where the Kubeconfig of the Seed cluster will be downloaded to.")
+	testShootsPrefix         = flag.String("prefix", "", "prefix to use for test shoots")
```

Comment:
can we make it possible to configure the actual name of the shoot from outside not just a prefix?
The Testmachinery calculates a unique shoot name so that we can also clean it up afterwards

Reply:
I think that would make sense for the create shoot test. 
I guess we can leave the prefix option for the scheduler and maintenance test that clean their shoots themselves.